infrastructure/cloud_run.py

```
@@ -4,6 +4,7 @@
 import project
 import topics
 import buckets
+import images
```

Ruff (0.11.9)

`images` imported but unused

Remove unused import: `images`

(F401)